packages/webapp/pages/account/profile.tsx

```diff
@@ -183,6 +209,17 @@ const AccountProfilePage = (): ReactElement => {
           leftIcon={<AtIcon />}
           value={user.username}
         />
+        {showExperienceLevel && (
+          <ExperienceLevelDropdown
+            defaultValue={experienceLevel as UserExperienceLevelKey}
+            name="experienceLevel"
+            className={{
+              container: 'mt-6 max-w-sm',
+              button:
+                '!text-text-primary hover:shadow-[inset_0.125rem_0_0_var(--theme-text-primary)]',
```

Comment:
Why is the override needed here? (it doesn't match guidelines?)

Reply:
You're right, not needed, removing...